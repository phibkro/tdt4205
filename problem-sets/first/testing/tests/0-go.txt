go
apple
go
banana
go
orange
