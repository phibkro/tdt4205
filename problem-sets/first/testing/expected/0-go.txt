Table filled!
line    1: accepted: go
line    2: error: apple
~~~~~~~~~~~~~~~~~~^
line    3: accepted: go
line    4: error: banana
~~~~~~~~~~~~~~~~~~^
line    5: accepted: go
line    6: error: orange
~~~~~~~~~~~~~~~~~~^
